modules/ROOT/pages/manage-exchange-api-task.adoc

```diff
@@ -29,11 +35,13 @@ Use this option if you created your API outside API Manager, but want to apply p
 Use this option if you want to provide additional security for your API. 
 +
 . Based on the type of endpoint you are creating, specify the application type for a basic endpoint or the proxy deployment target for an endpoint with proxy.
-. If you want to manage the API using Mule 4 or later, select the *Mule version* checkbox.
-> In *Implementation URI*, replace the example by entering the URL of the site that hosts the API.
+. If you want to manage the API using Mule 4 or later (recommended), accept the default selection.
 +
-For example, you can use the URL of your API asset in Exchange. In this case, the API is consuming the JSONPlaceholder API, so enter `+http://jsonplaceholder.typicode.com+`.
+The parameters that you must specify differs based on the Mule version you select. For detailed field descriptions, see <<field-reference,parameter details>>.
```

Comment:
"The parameters that you must specify differs" -> "The parameters that you must specify differ"

Reply:
Done! Thanks!